policies/social-media.md

```diff
@@ -1,4 +1,4 @@
-# Web of Things (WoT) Policy (Draft)
+# Web of Things (WoT) Policy
 Social Media (Twitter, LinkedIn, etc.) news requires quick publication. Thus, group resolutions would be too slow and a policy is required.
```

Comment:
It is just an "etc." so I would try to keep a generic list so that we do not need a policy later on. Even Twitter can change name to X, we can change to Mastadon so being not super specific is better here I think.

Reply:
ok, I see. Good point to rename Twitter to X or use "Twitter/X"